feat(ios): expose Ti.UI.Window.largeTitleAttributes

Max87ZA opened the pull request:
Use largeTitleAttributes similar to titleAttributes which applies custom style to window's title.
titleAttributes applies style to scrolled title(in navBar) only, largeTitleAttributes to big one only

Max87ZA committed:
kotlin 1.8.10, ios logging

hansemannn committed:
feat: add spring animations

hansemannn committed:
Merge branch 'master' into feature/ios17-spring-animations

Max87ZA committed:
Merge pull request #1 from tidev/master

merge tidev master into max87 kotlin_1810

Max87ZA committed:
network client fix

Max87ZA committed:
(fix):network client fix

Max87ZA committed:
Merge branch 'master' into kotlin_1810

Max87ZA committed:
Merge pull request #2 from Max87ZA/kotlin_1810

Kotlin 1810

Max87ZA committed:
feat(ios): largeTitleAttributes

Max87ZA committed:
feat(ios): largeTitleAttributes

Max87ZA committed:
feat(ios): largeTitleAttributes

hansemannn commented on android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java:
```diff
@@ -945,6 +945,14 @@ public void open(String method, String url)
+
+		// } else {
+		// 	Log.d(TAG, "Twitter: not sending X-Requested-With header", Log.DEBUG_MODE);
+		// }
```
Please remove the unused code

hansemannn commented on iphone/TitaniumKit/TitaniumKit/Sources/Modules/TiUIWindowProxy.m:
```diff
@@ -404,10 +405,10 @@ - (void)setTitleAttributes:(id)args
+      //      if ([self shouldUseNavBarApperance]) {
+      //        navigationBar.standardAppearance.largeTitleTextAttributes = theAttributes;
+      //        navigationBar.scrollEdgeAppearance.largeTitleTextAttributes = theAttributes;
+      //      }
```
Is there a reason you deactivated it here? Does it have an influence of "normal" large titles (without additional title styling)?

Max87ZA replied:
Yes, because originally titleAttributes applied style to both states(big and small), that condition is about - if largeTitleEnabled == true, apply style to both cases, else apply it to small one...

hansemannn commented on package.json:
```diff
@@ -1,7 +1,7 @@
   "name": "titanium-mobile",
   "description": "TiDev Titanium Mobile",
-  "version": "12.2.0",
+  "version": "12.2.2",
```
Please revert the version bump, as it's managed through the core

hansemannn reviewed (changes requested):
A few styling changes, nothing critical! Once changed, I can help with adding the docs. Finally, can you please follow [these steps](https://github.com/tidev/organization-docs/blob/main/AUTHORIZED_CONTRIBUTORS.md) to sign our contribution agreement? It's only necessary once, after that we can merge your changes much faster. Code strong! :)

Max87ZA committed:
fix(ios): removed commented code

m1ga commented on android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java:
```diff
@@ -945,6 +945,7 @@ public void open(String method, String url)
 
 		setReadyState(READY_STATE_OPENED);
 		setRequestHeader("User-Agent", TITANIUM_USER_AGENT);
+		
```
```suggestion
```

removing the new line so it won't show in this PR

m1ga committed:
Update android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java

Max87ZA committed:
Merge branch 'tidev:master' into master

Max87ZA committed:
Merge branch 'feat(ios)largeTitleAttributes'

m1ga commented:
@Max87ZA  I think you've merged https://github.com/tidev/titanium-sdk/pull/13891/ in your PR

Max87ZA committed:
fix(ios): removed setLargeTitleAttributes duplicity
fix(ios): removed bounce from TiAnimation
feat(android): Added collapse_toolbar props

Max87ZA commented:
Yeah, I removed it, because my local build failed on it... Now it builded fine

m1ga commented:
you still have some parts of the other PR in your PR and you've changed some Android layouts.

Max87ZA commented:
can you please discard this PR? I merged newest code from GA to my custom SDK, that's why there is android layout(about collapse toolbar)

m1ga commented:
sure, you can always close it (there is a button at the bottom) and create a new one. Or clean it up. Up to you :smile:
